src/components/Gnb.js

```diff
@@ -0,0 +1,51 @@
+import React, { useEffect, useState } from 'react';
+import Logo from './Logo';
+import CTA from './Cta';
+import { Link } from 'react-router-dom';
+import profileImg from '../assets/profile.svg';
+import { getProfile } from '../api/profile';
+import '../styles/gnb.css';
+function GNB() {
```

Comment:
보통 import 이 끝나면 한 줄 공백을 둡니다 

```suggestion
import '../styles/gnb.css';

function GNB() {
```